ingestion/src/metadata/config/workflow.py

```diff
@@ -114,3 +115,43 @@ def get_sink(
     logger.debug(f"Sink type: {sink_type}, {sink_class} configured")
 
     return sink
+
+
+def get_processor(
+    processor_type: str,
+    context: WorkflowContext,
+    processor_config: DynamicTypedConfig,
+    metadata_config: MetadataServerConfig,
+    _from: str = "ingestion",
+    **kwargs,
+) -> Processor:
+    """
+    Helps us to fetch and import the Processor class.
+
+    By default, we will pick it up from `ingestion`
+
+    We allow to pass any other specific object we may require.
+    E.g., for the ORM Profiler we need a Session to reach
+    the source tables.
+
+    :param processor_type: Type specified in the config, e.g., metadata-rest
+    :param context: Workflow related information
+    :param processor_config: Specific Processor configurations, such as the profiler and tests
+    :param metadata_config: Metadata server configurations
+    :param _from: From where do we load the sink class. Ingestion by default.
+    """
+    processor_class = get_class(
+        "metadata.{}.processor.{}.{}Processor".format(
```

Comment:
We need to make this more flexible to load the class without depending on the packaging convention. 
Not important right now but can be annoying for users to write their own sources, processors

Reply:
That's a good point. Let's tackle this together with https://github.com/open-metadata/OpenMetadata/issues/2848. I'll add a comment there